src/main/java/com/playmonumenta/networkrelay/BroadcastCommand.java

```diff
@@ -14,62 +19,95 @@
 import org.bukkit.event.Listener;
 import org.bukkit.plugin.Plugin;
 
-import dev.jorel.commandapi.CommandAPICommand;
-import dev.jorel.commandapi.CommandPermission;
-import dev.jorel.commandapi.arguments.GreedyStringArgument;
-
 public class BroadcastCommand implements Listener {
+	private static final List<NetworkRelayAPI.ServerType> ACCEPTED_SERVER_TYPES;
+
+	static {
+		List<NetworkRelayAPI.ServerType> acceptedServerTypes = new ArrayList<>();
+		acceptedServerTypes.add(NetworkRelayAPI.ServerType.ALL);
+		acceptedServerTypes.add(NetworkRelayAPI.ServerType.BUNGEE);
+		ACCEPTED_SERVER_TYPES = acceptedServerTypes;
+	}
```

Comment:
I think this can be
```
ACCEPTED_SERVER_TYPES = Collections.ListOf(NetworkRelayAPI.ServerType.ALL, NetworkRelayAPI.ServerType.MINECRAFT);
```

Reply:
Note also that this one has BUNGEE accepted, even though I think that's wrong / copy paste error?

Reply:
Yup, copy/paste error. I originally tried `List.of()`, but that wants to update Java first.

Reply:
Couldn't find a `Collections.ListOf()` or similar, but I found `Arrays.asList()`, which sounds familiar.